pavelib/paver_tests/test_servers.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=unicode-format-string
```

Comment:
Do we still need this? It looks like the strings were fixed in this file...

Reply:
not all the strings were fixed but regardless i'll remove for now